packages/samples/test/output/nested/openapi.json

```diff
@@ -7,7 +7,7 @@
   "tags": [],
   "paths": {
     "/sub/a/subsub": {
-      "get": {
+      "post": {
```

Comment:
The change of behavior here is that it was before using the following default
| Case                 | Before  | Now     |
| -------------------- | ------- | ------- |
| body with @body      | `@post` | `@post` |
| body with unannotted | `@get`  | `@post` |
| no body              | `@get`  | `@get`  |

Reply:
I think, if we are going to default operations to `@get`, then we should throw if there is a body or an unannotated parameter.  Per discussion, I believe we should not be doing operation guessing at all, and require specs using the openapi generators to specify verbs for all operations.  Similarly, an operation marked with `@get` should throw if there are any unannotated or body parameters, probably at the http level.

Reply:
on a related note, I see we still default the verb from the name of the operation, thought we agreed to remove that for now. Should I update this as well then?

Reply:
I think that's right, we should not have http verb inference in our openapi and code emitters, but I would want to be sure @bterlson is on the same page about this.  At the very least, if we are going to do any verb inference, it should eb simple, and should not change based on presence or absence of body (or unannotated) parameters.

Reply:
Yep, fine dropping inference based on operation name for now.